rock-n-rollup/src/services/operation.rs

```diff
@@ -151,9 +151,9 @@ where
 #[cfg(test)]
 mod tests {
 
-    use serde::Deserialize;
-
     use crate::core::{MockRuntime, Runtime, Service};
+    use serde::Deserialize;
+    //use tezos_smart_rollup_host::runtime::Runtime;
```

Comment:
this comment can also be removed

Reply:
it has been removed at this [commit](3d56b448ce27ff2ee25b9e46e3b86aecbbb74896)